gradlew

```diff
@@ -160,6 +160,9 @@ save () {
     echo " "
 }
 APP_ARGS=$(save "$@")
+# Realm's work-around for a bug in Gradle 4.1
```

Comment:
Also, we have more `gradlew` files than this, e.g. in `examples`, `realm-plugin`, `realm` and maybe more. Shouldn't this fix be in all of them?